store.go

```diff
@@ -1,3 +1,6 @@
+// Cloud Storage interface to make Local, Google, s3 file storage
```

Comment:
It does pull out from any file:

https://github.com/araddon/qlbridge/blob/master/datasource/memdb/db.go

https://godoc.org/github.com/araddon/qlbridge/datasource/memdb

Or maybe i don't understand, is there a good example?